translations/en.json

```diff
@@ -554,6 +554,7 @@
 				"rate": "24H funding rate"
 			},
 			"trade": {
+				"market": "market",
```

Comment:
Design was a bit inconsistent, but lets make this "Isolated Market"

Reply:
Done!